src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedVariantContextUtils.java

```diff
@@ -523,4 +523,217 @@ private static int getMinBaseQualityForAlleleInRead(final PileupElement pileupEl
         final OptionalInt minQuality = IntStream.range(0, pileupBaseQualities.length).map(i -> Byte.toUnsignedInt(pileupBaseQualities[i])).min();
         return minQuality.orElse(-1);
     }
+
```

Comment:
Can you put these new utility methods in `GATKVariantContextUtils` instead of `GATKProtectedVariantContextUtils`?

Reply:
Fixed!